gpMgmt/bin/gpdirtableload

```diff
@@ -355,17 +363,24 @@ class gpdirtableload:
         self.allFilesToDownload = []
         self.numFiles = 0
 
-        qry = "SELECT relative_path FROM %s " % self.options.table
+        qry = "SELECT relative_path FROM %s WHERE " % self.options.table
+
+        qry += " relative_path like \'%s" % self.options.input_file
+        if self.regexMatch:
+        	qry += '%'
+        qry += "\'"
 
         if self.options.tag:
-            qry += "WHERE tag = \'%s\'" % self.options.tag
+            qry += ", tag = \'%s\'" % self.options.tag
```

Comment:
The conditions should be joined by 'AND'. Do we have test cases for these options?

Reply:
Fixed,  as this tools is independent from kernel, does not add test cases now.